Formula/swift-protobuf.rb

```diff
@@ -0,0 +1,35 @@
+class SwiftProtobuf < Formula
+  desc "Plugin and runtime library for using protobuf with Swift"
+  homepage "https://github.com/apple/swift-protobuf"
+  url "https://github.com/apple/swift-protobuf/archive/0.9.904.tar.gz"
+  sha256 "d65970c3f17cb4a09e61102d7dce37733ad34fe723cf65795e856ec2cd126711"
+  revision 1
+  head "https://github.com/apple/swift-protobuf.git"
+
+  depends_on :xcode => ["8.3", :build]
+  depends_on "protobuf"
+
+  conflicts_with "protobuf-swift",
+    :because => "both install `protoc-gen-swift`"
+
+  def install
+    system "swift", "build", "-c", "release", "-Xswiftc", "-static-stdlib"
+    bin.install ".build/release/protoc-gen-swift"
```

Comment:
No man page, doc, or anything else?

Reply:
No man file, but I guess we could `doc.install`  [Documentation/PLUGIN.md](https://github.com/apple/swift-protobuf/blob/master/Documentation/PLUGIN.md) which explains what's how swift-protobuf is used.